src/App.js

```diff
@@ -1,9 +1,35 @@
 import "./App.css";
 
+// router
+import { Link, Routes, Route, } from "react-router-dom";
+
+// local components
+import Navbar from "./components/navbar/Navbar";
+import Footer from './components/footer/Footer'
+
+// pages
+import Signup from "./pages/signup/Signup";
+import Home from "./pages/home/Home";
+import Login from "./pages/login/Login";
+import Wishlist from "./pages/wishlist/Wishlist";
+import Product from "./pages/product/Product";
+import CartManagement from "./pages/cart-management/CartManagement";
+import PageNotFound from "./pages/pageNotFound/PageNotFound";
```

Comment:
You can create an index.js file and export all pages from there so that you can use barrel exports and this way your App.js file will look cleaner.

Reply:
this is a great suggestion, have made the changes 👍